talaiot/src/main/kotlin/com/cdsap/talaiot/publisher/InfluxDbPublisher.kt

```diff
@@ -35,6 +37,7 @@ class InfluxDbPublisher(
         } else {
             val url = "${influxDbPublisherConfiguration.url}/write?db=${influxDbPublisherConfiguration.dbName}"
             var content = ""
+            val thresholdConfiguration = influxDbPublisherConfiguration.threshold
```

Comment:
Could consider adding preconditional check to guard against negative values for `minExecutionTime ` and `maxExecutionTime`

Reply:
done